rll/metadata.rb

```diff
@@ -58,3 +51,9 @@
   :default => "cred:AWS_ACCESS_KEY_ID",
   :recipes => ["rll::test-script"]
 
+attribute   "DISABLE_AUTO_UPGRADE",
+  :display_name => "Disables auto upgrade if previously enabled",
```

Comment:
All updates done in [IV-957_iteration_rl10_upgrade branch] (https://github.com/rightscale/rightlink_scripts/tree/IV-957_iteration_rl10_upgrade)